src/pytest/__init__.py

```diff
@@ -46,8 +46,57 @@
 from _pytest.warning_types import PytestUnknownMarkWarning
 from _pytest.warning_types import PytestWarning
 
+# For mypy Any type checking purposes.
+# This file sets disallow_any_expr to ensure that the public API
+# does not have dynamic typing via Any. Manually using each public API
+# type as an expression to enforce this.
+__version__ = __version__
+register_assert_rewrite = register_assert_rewrite
+_setup_collect_fakemodule = _setup_collect_fakemodule
+cmdline = cmdline
+ExitCode = ExitCode
+# hookimpl = hookimpl
+# hookspec = hookspec
+main = main
+UsageError = UsageError
+__pytestPDB = __pytestPDB
+_fillfuncargs = _fillfuncargs
+fixture = fixture
```

Comment:
`fixture` and `yield_fixture` are currently untyped. Shouldn't it fail for them (for example)?